### PR TITLE
Create README-es.md

### PR DESCRIPTION
**Pull request by 4k1k0**
It's a spanish translation for the readme.

### CODE REVIEW

File: README-es.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<div align="center">
+  <a href="https://bcrikko.github.io/NES.css/" target="_blank"><img src="https://user-images.githubusercontent.com/5305599/49061716-da649680-f254-11e8-9a89-d95a7407ec6a.png" alt="NES.css: NES-style  CSS framework" style="max-width:100%;" width="600" height="315"></a>
+
+  <a href="README.md">English</a> / <a href="README-jp.md">日本語</a> / <a href="README-es.md">Español</a>
```
Comment (evexoio) [inline README-es.md]: Can you please add this line to `README.md` to add Spanish next to english and Japanese? 🙏

Comment (4k1k0) [reply README-es.md]: Sure thing. :smile:
It's done :octocat:

Comment (evexoio) [reply README-es.md]: Oops, I forgot to mention to add the same line to `README-jp.md` as well 🙈
Can you please do that? 😬

**Pull request merged**